duplicate.php

```diff
@@ -112,19 +140,22 @@ function edd_duplicate_post_taxonomies( $id, $new_id, $post_type ) {
 
 /**
  * Copy the meta information of a Product to another Product
+ *
+ * @param int $id     The original download ID.
+ * @param int $new_id The new download ID.
  */
-function edd_duplicate_post_meta($id, $new_id) {
+function edd_duplicate_post_meta( $id, $new_id ) {
 	global $wpdb;
-	$post_meta_infos = $wpdb->get_results("SELECT meta_key, meta_value FROM $wpdb->postmeta WHERE post_id=$id");
-
-	if (count($post_meta_infos)!=0) {
-		$sql_query = "INSERT INTO $wpdb->postmeta (post_id, meta_key, meta_value) ";
-		foreach ($post_meta_infos as $meta_info) {
-			$meta_key = $meta_info->meta_key;
-			$meta_value = addslashes($meta_info->meta_value);
-			$sql_query_sel[]= "SELECT $new_id, '$meta_key', '$meta_value'";
+	$post_meta_infos = $wpdb->get_results( $wpdb->prepare( "SELECT meta_key, meta_value FROM {$wpdb->postmeta} WHERE post_id = %d", $id ) );
+
+	if ( count( $post_meta_infos ) ) {
+		$sql_query = "INSERT INTO {$wpdb->postmeta} (post_id, meta_key, meta_value) ";
+		foreach ( $post_meta_infos as $meta_info ) {
+			$meta_key        = $meta_info->meta_key;
+			$meta_value      = addslashes( $meta_info->meta_value );
+			$sql_query_sel[] = "SELECT {$new_id}, '{$meta_key}', '{$meta_value}'";
```

Comment:
If you remove the `addslashes()` on `$meta_value` on line 155, then you should be able to replace this line with:

```suggestion
			$sql_query_sel[] = $wpdb->prepare(
				"SELECT %d, %s, %s",
				$new_id,
				$meta_key,
				$meta_value
			);
```